upup/pkg/kutil/delete_cluster.go

```diff
@@ -21,6 +21,9 @@ import (
 	"bytes"
```

Comment:
Should probably leave a deprecation message here if we're ok with this file being here temporarily